x-pack/plugins/triggers_actions_ui/public/application/lib/action_variables.ts

```diff
@@ -103,164 +107,31 @@ export enum SummaryAlertProvidedActionVariables {
-    description: i18n.translate('xpack.triggersActionsUI.actionVariables.ruleSpaceIdLabel', {
-      defaultMessage: 'The space ID of the rule.',
-    }),
-  },
-  [AlertProvidedActionVariables.ruleTags]: {
-    name: AlertProvidedActionVariables.ruleTags,
-    description: i18n.translate('xpack.triggersActionsUI.actionVariables.ruleTagsLabel', {
-      defaultMessage: 'The tags of the rule.',
-    }),
-  },
-  [AlertProvidedActionVariables.ruleType]: {
-    name: AlertProvidedActionVariables.ruleType,
-    description: i18n.translate('xpack.triggersActionsUI.actionVariables.ruleTypeLabel', {
-      defaultMessage: 'The type of rule.',
-    }),
-  },
-  [AlertProvidedActionVariables.ruleUrl]: {
-    name: AlertProvidedActionVariables.ruleUrl,
-    description: i18n.translate('xpack.triggersActionsUI.actionVariables.ruleUrlLabel', {
-      defaultMessage:
-        'The URL to the rule that generated the alert. This will be an empty string if the server.publicBaseUrl is not configured.',
-    }),
-    usesPublicBaseUrl: true,
-  },
-  [AlertProvidedActionVariables.date]: {
-    name: AlertProvidedActionVariables.date,
-    description: i18n.translate('xpack.triggersActionsUI.actionVariables.dateLabel', {
-      defaultMessage: 'The date the rule scheduled the action.',
-    }),
-  },
-  [AlertProvidedActionVariables.kibanaBaseUrl]: {
-    name: AlertProvidedActionVariables.kibanaBaseUrl,
-    description: i18n.translate('xpack.triggersActionsUI.actionVariables.kibanaBaseUrlLabel', {
-      defaultMessage:
-        'The configured server.publicBaseUrl value or empty string if not configured.',
-    }),
-  },
-};
-
-function prefixKeys(actionVariables: ActionVariable[], prefix: string): ActionVariable[] {
-  return actionVariables.map((actionVariable) => {
-    return { ...actionVariable, name: `${prefix}${actionVariable.name}` };
-  });
-}
+type ActionVariablesWithoutName = Omit<ActionVariable, 'name'>;
 
-// this list should be the same as in:
-//   x-pack/plugins/alerting/server/task_runner/transform_action_params.ts
-function getAlwaysProvidedActionVariables(): ActionVariable[] {
-  const result: ActionVariable[] = [];
-
-  result.push(AlertProvidedActionVariableDescriptions[AlertProvidedActionVariables.ruleId]);
-
-  result.push(AlertProvidedActionVariableDescriptions[AlertProvidedActionVariables.ruleName]);
-
-  result.push(AlertProvidedActionVariableDescriptions[AlertProvidedActionVariables.ruleSpaceId]);
-
-  result.push(AlertProvidedActionVariableDescriptions[AlertProvidedActionVariables.ruleTags]);
-
-  result.push(AlertProvidedActionVariableDescriptions[AlertProvidedActionVariables.ruleType]);
-
-  result.push(AlertProvidedActionVariableDescriptions[AlertProvidedActionVariables.ruleUrl]);
-
-  result.push(AlertProvidedActionVariableDescriptions[AlertProvidedActionVariables.date]);
-
-  result.push({
-    name: AlertProvidedActionVariables.alertId,
+const AlertProvidedActionVariableDescriptions: Record<
```

Comment:
Im wondering if we should move this to a package like `alerting-browser` or something since my rule form needs these variables as well. If not I can move them in a separate PR

Reply:
I would like to avoid this work on this PR. I treat this PR as a bug fix, not a feature implementation. Also, we need to discuss and standardize the naming of our packages, a process that can slow this PR. Do you mind if you do it on your PR?

Reply:
All good